CODEOWNERS

```diff
@@ -6,3 +6,7 @@
 /packages/plugins/*_acd/ @theforeman/packaging @theforeman/atix
 /packages/plugins/*_resource_quota/ @theforeman/packaging @theforeman/atix
 /packages/plugins/*_snapshot_management/ @theforeman/packaging @theforeman/atix
+/packages/plugins/*_salt/ @theforeman/packaging @theforeman/salt
+/packages/plugins/*_puppet/ @theforeman/packaging @theforeman/puppet
+/packages/katello/*_kernel_care/ @theforeman/packaging @theforeman/foreman_kernel_care
+/packages/plugins/*_fog_proxmox/ @theforeman/packaging @theforeman/proxmox
```

Comment:
We also have `packages/plugins/rubygem-fog-proxmox`, should this be part of the permission?

Reply:
The proxmox team does not own those https://github.com/orgs/theforeman/teams/proxmox/repositories so I would say no.